sys/include/net/ng_netreg.h

```diff
@@ -126,6 +126,30 @@ int ng_netreg_num(ng_nettype_t type, uint32_t demux_ctx);
  */
 ng_netreg_entry_t *ng_netreg_getnext(ng_netreg_entry_t *entry);
 
+/**
+ * @brief   Calculates the checksum for a packet of type @p checksum_hdr_type.
+ *
+ * @param[in] pseudo_hdr_type   The type of the pseudo header if required.
+ *                              May be NG_NETTYPE_UNDEF if no pseudo header is
+ *                              required for the checksum calculation.
```

Comment:
You could be more consistent with the use of periods at the end of the comments -> I would omit them...